apis/expenses-api-express/v1.0/README.md

```diff
@@ -0,0 +1,56 @@
+![Auth0 Open Source Software](https://cdn.auth0.com/resources/oss-source-large-2x.png)
+
+# Expenses API Express - The Expenses API is a simple api that is intended to be used as a starting point for hands on labs.
```

Comment:
Instead of saying this is a starting point, we could reword this so that it reads like an example application that implements oAuth 2 API protection.

Reply:
Also the hash here makes this line the title of the document, we should add some returns to the description so that it is normal text.